org.rdkit.knime.nodes/src/org/rdkit/knime/nodes/molextractor/RDKitMoleculeExtractorNodeModel.java

```diff
@@ -1,809 +1,810 @@
-						// Create a data row
-						final DataRow rowNew = new DefaultRow("row_" + rowOutputIndex,
-								createResultCells(RDKitMolCellFactory.createRDKitAdapterCell(listFragments.get(iFrag)), cellRef));
-						newTableData.addRowToTable(rowNew);
-						rowOutputIndex++;
-					}
-				}
-			}
-			catch (final Exception exc) {
-				boolean bSkip = false;
-				switch (errorHandling) {
-				case Fail:
-					throw new Exception("Input molecule could not be processed due to a fragmentation error.");
-
-				case SkipWithWarning:
-					bSkip = true;
-					// Fall through
-
-				case MissingCellWithWarning:
-					getWarningConsolidator().saveWarning(WarningConsolidator.ROW_CONTEXT.getId(),
-							"Input molecule could not be processed due to a fragmentation error.");
-					break;
-
-				case SkipWithoutWarning:
-					bSkip = true;
-					break;
-				
-				case MissingCellWithoutWarning:
-				default:
-					break;
-				}
-
-				if (!bSkip) {
-					// Create a empty data row
-					final DataRow rowNew = new DefaultRow("row_" + rowOutputIndex,
-							createResultCells(cellOrig, cellRef));
-					newTableData.addRowToTable(rowNew);
-					rowOutputIndex++;
-				}
-			}
-		}
-		else {
-			// Create a data row
-			final DataRow rowNew = new DefaultRow("row_" + rowOutputIndex,
-					createResultCells(cellOrig, cellRef));
-			newTableData.addRowToTable(rowNew);
-			rowOutputIndex++;
-		}
-
-		return rowOutputIndex;
-	}
-
-	/**
-	 * Returns true, if the node is setup to produce a reference column in the output table.
-	 * Unless the model for the reference input role is set to "None" (null and does not use row ID)
-	 * this returns true.
-	 * 
-	 * @return True or false.
-	 */
-	private boolean isProducingReferenceColumn() {
-		return (m_modelReferenceInputColumnName.getColumnName() != null || m_modelReferenceInputColumnName.useRowID());
-	}
-
-	/**
-	 * Creates an array of results cells.
-	 * 
-	 * @param molCell Mol cell to be always included.
-	 * @param refCell Reference to be included only, if it is not null.
-	 * 
-	 * @return DataCell array.
-	 */
-	private DataCell[] createResultCells(DataCell molCell, final DataCell refCell) {
-		if (molCell == null) {
-			molCell = DataType.getMissingCell();
-		}
-		if (refCell == null) {
-			return new DataCell[] { molCell };
-		}
-		return new DataCell[] { molCell, refCell };
-	}
-
-	//
-	// Static Public Methods
-	//
-
-	/**
-	 * Determines, if the condition is fulfilled that we have a molecules input table with
-	 * columns connected to the node according to the passed in specs.
-	 * 
-	 * @param inSpecs Port specifications.
-	 * 
-	 * @return True, if there is a molecules table present at the first index of the specs,
-	 * 		and if it has columns.
-	 */
-	public static boolean hasMoleculesInputTable(final PortObjectSpec[] inSpecs) {
-		return (inSpecs != null && inSpecs.length >= 1 &&
-				inSpecs[0] instanceof DataTableSpec &&
-				((DataTableSpec)inSpecs[0]).getNumColumns() > 0);
-	}}
+/*
```

Comment:
Yes, I noticed that, too. But it was already too late. Would you know how to convert the entire project once to have the correct line endings and encodings for all Java classes? I would be grateful if you as a senior Git user could do that to avoid any mistakes.